gammapy/spectrum/models.py

```diff
@@ -226,14 +227,14 @@ def from_dict(cls, val):
         return model
 
     def plot(
-        self,
-        energy_range,
-        ax=None,
-        energy_unit="TeV",
-        flux_unit="cm-2 s-1 TeV-1",
-        energy_power=0,
-        n_points=100,
-        **kwargs
+            self,
```

Comment:
There's a lot of unrelated lines that show up in this diff. Could you please run `black gammapy/spectrum/models.py gammapy/spectrum/tests/test_models.py` to go back to the black coding style?